core/src/main/java/com/hamdan/forzenbook/core/Entry.kt

```diff
@@ -0,0 +1,6 @@
+package com.hamdan.forzenbook.core
+
+data class Entry(
```

Comment:
This name is generic, `Entry`, but then it includes `LoginError` which isn't generic. And is this used JUST for login errors? If this is to be reusable don't tie it tight to Login error

Reply:
Made a generic Error interface which LoginError now implements, made the data class Entry have its error field changed of type LoginError to Error.